fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -577,6 +577,11 @@ rocksdb::ColumnFamilyOptions getCFOptions() {
 }
 
 rocksdb::ColumnFamilyOptions getCFOptionsForInactiveShard() {
+	if (g_network->isSimulated()) {
+		return getCFOptions();
+	} else {
+		ASSERT(false); // FIXME: remove when SHARDED_ROCKSDB_DELAY_COMPACTION_FOR_DATA_MOVE feature is well tuned
```

Comment:
So this function should never be invoked on real cluster yet?

Reply:
This function is expecting to improve compaction efficiency when data moves. However, currently, it is experimental. So, this feature should not be invoke in real clusters for now.